src/components/Api.js

```diff
@@ -3,12 +3,13 @@ export const instance = create({
   baseURL: `${process.env.REACT_APP_API_BASE_URL}`,
 });
 export const getBeersService = async (filters, currentPage) => {
-  const { id, beerName, yeast, date, abv } = filters;
+  const { id, beerName, yeast, date, abv, postPerPage } = filters;
   const idQueryString = id ? `&ids=${id}` : "";
   const yeastQueryString = yeast ? `&yeast=${yeast}` : "";
   const beerQueryString = beerName ? `&beer_name=${beerName}` : "";
   const dateQueryString = date ? `&brewed_after=${date}` : "";
   const abvQueryString = abv ? `&abv_gt=${abv}` : "";
-  const url = `/beers?page=${currentPage}&per_page=${filters.noOfPost}${dateQueryString}${abvQueryString}${beerQueryString}${idQueryString}${yeastQueryString}`;
+  const PerPageQueryString = postPerPage ? `&per_page=${postPerPage}` : "";
```

Comment:
What is API response if we don't pass per_page in the url string?
If it doesn't return any thing or fails then need to handle that as well.

Reply:
if we don't pass per_page in the url string then the per_page default value will 25 post per page and it will not fail